<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client_Telemetry_Info_Form</class>
 <widget class="QDialog" name="Client_Telemetry_Info_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>156</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>156</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_telemetry">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>单一遥测信息</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lab_addr">
           <property name="text">
            <string>遥测地址</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_inputFlag">
           <property name="text">
            <string>inputFlag</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_dataType">
           <property name="text">
            <string>数据类型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_describe">
           <property name="text">
            <string>描述</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_addr"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_inputFlag"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dataType"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_describe"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
